index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="index2.css" />
+  </head>
+  <body>
+    <section class="first-section">
+      <div class="first-container">
+        <nav class="first-nav">
+          <h1 class="my-logo">My Logo</h1>
```

Comment:
- [ ] [REQUIRED] I suggest that you replace the `<h1>` element with an `<a>` element ♻️. It is going to make the Logo a clickable item and this will give the user 👨 a more intuitive experience, since roughly most websites on the internet 🌐 got the Logo clickable.